src/nisarqa/processing/caltools/abscal.py

```diff
@@ -170,6 +170,38 @@ def create_dataset_from_abscal_results(
         ds_dtype=np.bytes_,
     )
 
+    create_dataset_from_abscal_results(
+        key="latitude_deg",
+        ds_name="latitude",
+        ds_descr=(
+            "The geodetic latitude of the corner reflector at the time it was"
+            " surveyed."
+        ),
+        ds_dtype=np.float64,
+        ds_units="degree_north",
+    )
+
+    create_dataset_from_abscal_results(
+        key="longitude_deg",
+        ds_name="longitude",
+        ds_descr=(
+            "The longitude of the corner reflector at the time it was surveyed."
+        ),
+        ds_dtype=np.float64,
+        ds_units="degree_east",
+    )
+
+    create_dataset_from_abscal_results(
+        key="height_above_ellipsoid",
+        ds_name="heightAboveEllipsoid",
+        ds_descr=(
+            "The height of the corner reflector above the WGS 84 reference"
```

Comment:
Even though "WGS 84" (with a space) is correct, let's remove the space for consistency with similar descriptions in other NISAR products.

```suggestion
            "The height of the corner reflector above the WGS84 reference"
```